service/src/test/java/uk/gov/mca/beacons/api/beaconsearch/BeaconSpecificationSearchServiceUnitTest.java

```diff
@@ -140,6 +142,111 @@ void givenEmailAndAccountHolder_ThenShouldCallRepositoryWithSpecificationAndSort
     );
   }
 
+  @Test
+  public void shouldCallRepositoryWithSpecificationOnlyForFullExport() {
```

Comment:
Looking at the assertions, I'm not sure the name of the test describes what is being tested here?